docs/vars.rst

```diff
@@ -259,6 +259,25 @@ For the PyMuPDF programmer, some combination (using Python's `|` operator, or si
 
     2048 -- Ignore built-in differences between text appearing in e.g. PDF viewers versus text stored in the PDF. See :ref:`AdobeManual`, page 615 for background. If set, the **stored** ("replacement" text) is ignored in favor of the displayed text.
 
+.. py:data:: TEXT_COLLECT_STRUCTURE
+
+    256 -- Not supported.
+
+.. py:data:: TEXT_ACCURATE_BBOXES
+
+    512 -- Calculates exact bboxes for each glyph, instead of relying on the information in the font or PDF document.
+
+.. py:data:: TEXT_COLLECT_VECTORS
+
+    1024 -- 
```

Comment:
Should we have a comment against this number?

Reply:
Ah yes, good point. I've marked as `Not supported.` in my tree. Will push to this PR just prior to merging after tests pass.